src/blockchain/log-processors/crowdsourcer.ts

```diff
@@ -2,42 +2,54 @@ import Augur from "augur.js";
 import * as Knex from "knex";
 import { FormattedEventLog, ErrorCallback } from "../../types";
 import { augurEmitter } from "../../events";
+import { insertPayout } from "./database";
 
 export function processDisputeCrowdsourcerCreatedLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedEventLog, callback: ErrorCallback): void {
-  console.log("TODO: DisputeCrowdsourcerCreated");
-  console.log(log);
-  callback(null);
+  insertPayout( db, trx, log.market, log.payoutNumerators, log.invalid, (err, payoutID) => {
+    const crowdsourcerToInsert = {
+      blockNumber: log.blockNumber,
+      transactionHash: log.transactionHash,
+      logIndex: log.logIndex,
+      crowdsourcerID: log.disputeCrowdsourcer,
+      marketID: log.market,
+      size: log.size,
+      payoutID,
+    };
+    db.transacting(trx).insert(crowdsourcerToInsert).into("crowdsourcers").returning("crowdsourcerID").asCallback(callback);
+    augurEmitter.emit("DisputeCrowdsourcerCreated", log);
```

Comment:
Do we want these Emitted before the DB write completes?

I think it works, but worth a double check there's no funny race condition. Main issue I see is if that insert fails we've emitted an event that isn't look-up-able in the DB.

Reply:
You're right, I was being lazy, fixing